c/validation/adbc_validation.cc

```diff
@@ -911,6 +911,167 @@ void ConnectionTest::TestMetadataGetObjectsPrimaryKey() {
   ASSERT_EQ(constraint_column_name, "id");
 }
 
+void ConnectionTest::TestMetadataGetObjectsForeignKey() {
+  ASSERT_THAT(AdbcConnectionNew(&connection, &error), IsOkStatus(&error));
+  ASSERT_THAT(AdbcConnectionInit(&connection, &database, &error), IsOkStatus(&error));
+
+  if (!quirks()->supports_get_objects()) {
+    GTEST_SKIP();
+  }
+
+  std::optional<std::string> maybe_parent_ddl =
+      quirks()->ForeignKeyParentTableDdl("adbc_fkey_parent_test");
```

Comment:
```suggestion
      quirks()->PrimaryKeyTableDdl("adbc_fkey_parent_test");
```
To match the previous suggestion